### PR TITLE
Set default storage from containers.conf for temporary images

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/11107

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ require (
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.9.1
 	github.com/containers/buildah v1.22.1-0.20210824095355-fc1e256f5051
-	github.com/containers/common v0.44.0
+	github.com/containers/common v0.44.1-0.20210914173811-fcaa2e0de285
```
Comment (TomSweeneyRedHat) [inline go.mod:15]: If this is heade towards RHEL 8.5.0.2, we probably ought to cut a common v0.44.1

**Comment by TomSweeneyRedHat** [comment]
A question on the versioning of c/common, otherwise LGTM

File: go.mod (hunk 1 above)
Comment (rhatdan) [reply go.mod:15]: Will cut a release for podman 3.4.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Pull request merged**